.github/workflows/persubmit.yml

```
@@ -7,12 +7,21 @@ jobs:
   build_and_test:
     name: Build and Test
     runs-on: ubuntu-latest
-    if: ${{ !contains(github.event.pull_request.labels.*.name, 'skip ci') }}
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'skip ci') || contains(github.event.head_commit.message, '[skip ci]' }}
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8]
+        #with_opengl: [0, 1]
+        #exclude:
+        #  - python: 3.6
+        #    with_opengl: ubuntu-16.04
+        #  - python: 3.8
+        #    with_opengl: ubuntu-16.04
```

```suggestion
        #with_opengl: [0, 1]
        #exclude:
        #  - python: 3.6
        #    with_opengl: ubuntu-16.04
        #  - python: 3.8
        #    with_opengl: ubuntu-16.04
```

Any reason to keep these comments?

---

Wanna implement this for testing both gpu and cpu, fine, let's do it iapr.

---

Oh, that would be nice! The 6 matrices will run in parallel, right?